checker/tests/toy/TestDefaultForTypeDecl.java

```diff
@@ -0,0 +1,5 @@
+import org.checkerframework.checker.toy.qual.*;
+
+public class TestDefaultForTypeDecl {
+    void test(@ToyTop TestDefaultForTypeDecl arg) {}
```

Comment:
Could you expand the test a bit to show that normal uses of the type work, e.g. unannotated uses and with other qualifiers.